reviews/forms.py

```diff
@@ -116,10 +113,16 @@ def __init__(self, *args, **kwargs):
             "allow_long_route_continuation", False
         )
         super(ReviewCloseForm, self).__init__(*args, **kwargs)
+        self.fields["continuation"].choices = [
+            x
+            for x in Review.Continuations.choices
+            if x[0] != Review.Continuations.DISCONTINUED
+        ]
+
         if not allow_long_route_continuation:
             self.fields["continuation"].choices = [
                 x
-                for x in Review.Continuations.choices
+                for x in self.fields["continuation"].choices
                 if x[0] != Review.Continuations.LONG_ROUTE
             ]
```

Comment:
1. I think it would be good to have a comment along the lines of `# Reviews should only be discontinued with ReviewDiscontinueView`
2. (optional) This may be a question of taste but two multiline list comprehensions just to remove two options is a bit awkward. An exclude list or a solution with `pop()` would look nicer.